### PR TITLE
Rename endpoints for ONT mutations

### PR DESCRIPTION
**Pull request by sdjm-tessella**
Confirmed as working by Me, Chris and Steve

### CODE REVIEW

File: src/graphql/queries/CreatePlateWithSamples.mutation.gql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 mutation ($barcode: String!, $wells: [WellWithSampleInput!]!) {
-  createPlateWithCovidSamples( 
+  createPlateWithSamples(
```
Comment (Chris-Friend) [inline src/graphql/queries/CreatePlateWithSamples.mutation.gql:2]: We've gone for `createPlateWithSamples` here...

File: src/graphql/queries/PoolSamples.mutation.gql [hunk 2]
```diff
@@ -1,5 +1,5 @@
 mutation ($plate_barcode: String!) {
-  createCovidLibraries (
+  createOntLibraries (
```
Comment (Chris-Friend) [inline src/graphql/queries/PoolSamples.mutation.gql:2]: ...but `createOntLibraries` here. I'm wondering why these are different? Both creations are in the Ont pipeline

Comment (sdjm-tessella) [reply src/graphql/queries/PoolSamples.mutation.gql:2]: It was a decision I made based on the inputs.  While plates are not Ont specific in themselves and you can pass in an ambiguous set of inputs, the OntLibraries endpoint needs you to tell it which plate you're going to create libraries for and that plate needs to have samples of a specific type on them.  It was somewhat arbitrary, but it felt like in the former mutation you should specify which type of samples you want and there's no way to confirm which type you sent from just the input hash.  In the latter case, you can confirm that the plate is of the right type.

Comment (Chris-Friend) [reply src/graphql/queries/PoolSamples.mutation.gql:2]: And that is why the GraphQL create plates mutation now includes a study type argument?

**Pull request merged**